themes-default/slim/yarn.lock

```diff
@@ -3188,6 +3187,10 @@ des.js@^1.0.0:
     inherits "^2.0.1"
     minimalistic-assert "^1.0.0"
 
+desandro-matches-selector@^2.0.0:
```

Comment:
`yarn.lock` shouldn't be modified.
Do you want me to add it to the travis check, so it would fail if you committed an unnecessary change to `yarn.lock`?

Reply:
Okay will do that